src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -71,7 +72,32 @@ public SyntaxTree SyntaxTree
         /// <returns></returns>
         public IOperation GetOperation(SyntaxNode node, CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.GetOperationCore(node, cancellationToken);
+            if (this.Compilation.IsIOperationFeatureEnabled())
```

Comment:
this should be inverted.  check if teh feature is not enabled, and immediately throw if so.  The throw for this check should not be at the bottom of this method.